src/geometry/manifold/algebra/lie_group.lean

```diff
@@ -115,6 +115,18 @@ lemma smooth_on.div {f g : M → G} {s : set M}
   (hf : smooth_on I' I f s) (hg : smooth_on I' I g s) : smooth_on I' I (f / g) s :=
 by { rw div_eq_mul_inv, exact ((smooth_mul I).comp_smooth_on (hf.prod_mk hg.inv) : _), }
 
+-- todo: many variants
```

Comment:
could you add the variants in this PR (with `within_at`, `on`, and so on)? This should be pretty mechanical.